lib/miq_automation_engine/models/miq_ae_datastore/xml_yaml_converter.rb

```diff
@@ -2,7 +2,8 @@ module MiqAeDatastore
   class XmlYamlConverter
     def self.convert(xml, domain_name, export_options)
       temp_domain = MiqAeDatastore.temp_domain.downcase
-      MiqAeDomain.create!(:name => temp_domain, :priority => 100, :enabled => false)
+      MiqAeDomain.create!(:name   => temp_domain, :priority => 100, :enabled => false,
+                          :tenant => Tenant.root_tenant)
```

Comment:
@kbrock 
This is for legacy XML models that we are trying to convert to YAML. We create a temporary domain under the root tenant import the old XML, export it out as YAML and delete the temporary domain.